Dockerfile

```diff
@@ -1,17 +1,17 @@
-# Use the official Python 3.10 image as the base image
-FROM python:3.11
+# Use the official Python 3.11 image as the base image
+FROM python:3.11-slim
```

Comment:
your comment should explain why you've chosen -slim (as it is the comment is not useful in itself and the current content can be removed as it doesn't add information to the line of code we can read below the comment)

Reply:
done